src/components/ProjectSearch/ProjectSearch.js

```diff
@@ -87,23 +117,30 @@ class ProjectSearch extends Component {
     this.props.toggleProjectSearch(false);
   }
 
+  renderFileSearch() {
```

Comment:
should this be used in render? it looks like it has logic that was moved out of the render function